src/main/java/org/folio/circulation/services/LostItemFeeRefundService.java

```diff
@@ -203,8 +210,47 @@ private static boolean shouldSchedulePatronNotices(LostItemFeeRefundContext cont
     final Result<CqlQuery> fetchQuery = exactMatch("loanId", context.getLoan().getId())
       .combine(exactMatchAny("feeFineType", feeFineTypes), CqlQuery::and);
 
-    return accountRepository.findAccountsAndActionsForLoanByQuery(fetchQuery)
-      .thenApply(r -> r.map(context::withAccounts));
+      return accountRepository.findAccountsAndActionsForLoanByQuery(fetchQuery)
+        .thenApply(r -> r.map(this::filterAccountsForRefund))
+        .thenApply(r -> r.map(context::withAccounts));
+  }
+
+  private Collection<Account> filterAccountsForRefund(Collection<Account> accounts) {
+    Account latestLostItemFeeAccount = getLatestAccount(accounts, LOST_ITEM_FEE_TYPE);
+
+    if (latestLostItemFeeAccount != null
+      && latestLostItemFeeAccount.getPaymentStatus() != null
+      && latestLostItemFeeAccount.getCreationDate() != null
+      && !latestLostItemFeeAccount.getPaymentStatus().contains(CANCELLED_PAYMENT_STATUS)) {
+
+      Collection<Account> filteredAccounts = new ArrayList<>();
+      filteredAccounts.add(latestLostItemFeeAccount);
+      DateTime creationDate = latestLostItemFeeAccount.getCreationDate();
+      Account latestLostItemFeeProcessingAccount = getLatestAccount(accounts,
+        LOST_ITEM_PROCESSING_FEE_TYPE);
+
+      if (latestLostItemFeeProcessingAccount != null
+        && !latestLostItemFeeAccount.getPaymentStatus().contains(CANCELLED_PAYMENT_STATUS)
+        && isDifferenceOneMinuteOrLess(creationDate, latestLostItemFeeAccount)) {
+
+        filteredAccounts.add(latestLostItemFeeProcessingAccount);
+      }
+      return filteredAccounts;
+    }
+
+    return accounts;
+  }
+
+  private boolean isDifferenceOneMinuteOrLess(DateTime creationDate, Account account) {
+    return Math.abs(minutesBetween(creationDate, account.getCreationDate()).getMinutes()) <= 1;
+  }
+
+  private Account getLatestAccount(Collection<Account> accounts, String lostItemFeeType) {
```

Comment:
This method can potentially work with any `feeFineType`, not just Lost Item (Processing) Fees. So please consider renaming last parameter:
```suggestion
  private Account getLatestAccount(Collection<Account> accounts, String feeFineType) {
```